temporal-testing-junit4/src/main/java/io/temporal/testing/internal/SDKTestWorkflowRule.java

```diff
@@ -52,18 +53,21 @@
 import java.time.Duration;
 import java.util.ArrayList;
 import java.util.List;
-import java.util.concurrent.ExecutionException;
-import java.util.concurrent.ScheduledExecutorService;
-import java.util.concurrent.ScheduledFuture;
-import java.util.concurrent.ScheduledThreadPoolExecutor;
-import java.util.concurrent.TimeUnit;
+import java.util.concurrent.*;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import org.junit.Test;
 import org.junit.rules.TestRule;
+import org.junit.rules.Timeout;
 import org.junit.runner.Description;
 import org.junit.runners.model.Statement;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class SDKTestWorkflowRule implements TestRule {
```

Comment:
Docstring about why you would use this over normal TestWorkflowRule would be useful

Reply:
It's in internal package, it's for us (SDK devs) only

Reply:
Right, still useful for new devs taking a look at it I mean.